web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -1404,6 +1404,25 @@ fixmystreet.assets.shropshire.streetlight_asset_message = function(asset) {
     return out;
 };
 
+/* Surrey */
+
+fixmystreet.assets.surrey = {};
+
+fixmystreet.assets.surrey.road_not_found = function(asset) {
+    var permissive_categories = ['Flooding inside a building', 'Flooding not on the highway but in a park or private land'];
```

Comment:
Just thinking, could we include this as data on the layer?  (The argument to this function is the layer, not an asset). eg if we store it in `permissive_categories` in the data, it should be lookable here as layer.fixmystreet.permissive_categories.